CMakeLists.txt

```diff
@@ -1,18 +1,24 @@
 # This is the main CMake file for NCEPLIBS-wrf_io.
 #
-# 
+# Other programmer, Ed Hartnett
 cmake_minimum_required(VERSION 3.15)
 
+# Get the version from the VERSION file.
 file(STRINGS "VERSION" pVersion)
 
-project(
-  wrf_io
-  VERSION ${pVersion}
-  LANGUAGES Fortran)
+# This provides cmake_print_variables() function for debugging.
+include(CMakePrintHelpers)
 
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/CMakeModules/Modules")
```

Comment:
What's wrong with list Append? Other than the path obviously

Reply:
It's not needed any more.

Reply:
It is not needed because of the set statement on line 13.
The question was, why is set better than list append?

Reply:
It's not particularly, I just cut and pasted. But it's not any worse either. ;-)